components/kyma-environment-broker/internal/process/provisioning/initialisation.go

```diff
@@ -82,6 +84,12 @@ func (s *InitialisationStep) Run(operation internal.ProvisioningOperation, log l
 	switch {
 	case err == nil:
 		operation.InputCreator = creator
+
+		err := s.updateInstance(operation.InstanceID, creator.Provider())
```

Comment:
nit: I'm wondering if we need to update the instance here, can we just update in the `create_runtime` step where `instanceStorage` already there?
Or if it's possible directly in `broker/instance_create` by using `HyperscalerTypeForPlanID` from `/common/hyperscaler/account_provider.go` file